backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -92,10 +91,24 @@ console: instance/etc/zope.ini ## Start a console into a Plone instance
 create-site: instance/etc/zope.ini ## Create a new site from scratch
 	PYTHONWARNINGS=ignore $(BIN_FOLDER)/zconsole run instance/etc/zope.conf ./scripts/create_site.py
 
-.PHONY: check
-check: $(BIN_FOLDER)/tox ## Check and fix code base according to Plone standards
+# QA
+.PHONY: lint
+lint: $(BIN_FOLDER)/uv ## Check and fix code base according to Plone standards
+	@echo "$(GREEN)==> Lint codebase$(RESET)"
+	@uvx ruff@latest check --fix --config $(BACKEND_FOLDER)/pyproject.toml
+	@uvx pyroma@latest -d .
+	@uvx check-python-versions@latest .
+	@uvx zpretty@latest --check src
```

Comment:
Do we want to limit the files to `--include="\.xml$"`?

Reply:
Nice to have: split these into separate make commands. Then the GHA can run them instead of duplicating the commands between Makefile and GHA config. And they're available in case the developer really just wants to run one tool.

Reply:
Make sure you test this without `uv` installed. I think you need to use `$(BIN_FOLDER)/uvx` everywhere.